geonode/harvesting/harvesters/base.py

```diff
@@ -127,20 +156,56 @@ def update_harvesting_session(
 
     def update_geonode_resource(
             self,
-            resource_descriptor: resourcedescriptor.RecordDescription,
+            harvestable_resource_id: int,
```

Comment:
Why did you change this method's signature? The issue description mentioned that the implementation of the rest of the code that calls this method is already done, please don't go off changing it. If you have a good reason for it, please justify it, otherwise please revert this back to the original implementation, where the extraction of a remote server's resource is the responsibility of the `get_resource()` method and the responsibility of creating a resource on the local GeoNode is the responsibility of the `update_geonode_resource()` method.

Reply:
Updated @ricardogsilva